resources/defaults.rb

```diff
@@ -2,7 +2,7 @@
 
 BASE_COMMAND = '/usr/bin/defaults'.freeze
 
-property :domain, String, name_property: true
+property :domain, String, name_property: true, coerce: proc { |m| m.include?(' ') ? Shellwords.shellescape(m) : m }
```

Comment:
Any reason this needs to be here rather than where we use this property? Also it seems if it's safe to `shellescape` this property when it has spaces, it's safe to `shellescape` it all the time, no?

Reply:
This block seems unnecessary; `new_resource.domain.shellescape` on line 25 would do the same thing, right?